### PR TITLE
FKM-14 | Wrong display of PDF files.

### PR DESCRIPTION
**Pull request by sergey347**

### CODE REVIEW

File: drupal/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.field.inc [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+/**
+ * Implements hook_field_formatter_info().
+ */
+function favrskov_helper_field_formatter_info() {
+  $info['favrskov_file_download_link'] = [
+    'label' => t('Favrskov Download link'),
+    'description' => t('Displays a link that will force the browser to download the file.'),
+    'field types' => ['file', 'image'],
+    'settings' => [
+      'text' => t('[file:name]'),
```
Comment (temoor-gilmutdinov) [inline drupal/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.field.inc]: There's no need to wrap token in t().

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/62/

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/63/

File: drupal/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.module [hunk 2]
```diff
@@ -5,6 +5,8 @@
  * Main file of favrskov_helper.module
  */
 
+require_once dirname(__FILE__) . '/favrskov_helper.field.inc';
```
Comment (temoor-gilmutdinov) [inline drupal/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.module]: It is better to include additional files through .info file.

Comment (temoor-gilmutdinov) [inline drupal/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.module]: Actually *.file.inc are not supported properly, comment above can be disregarded.

**Comment by ffw-bot-ua** [comment]
Build comment file: 
--------------
===============

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.cibox03.m1.propeople.com.ua/job/PR_BUILDER/64/

**Pull request merged**